functions/exercise19/scriptSpec.js

```diff
@@ -0,0 +1,44 @@
+/// <reference path="../../typings/globals/jasmine/index.d.ts" />
+/*globals returnElementsLargerThanNumber*/
+
+describe("Returning array elements larger than a number", function () {
+    it("empty array aren't any elements larger than 10", function () {
+        var elementsLargerThanANumber = returnElementsLargerThanNumber(10, []);
+        expect(elementsLargerThanANumber).toEqual([]);
+    });
+    it("in the array whit white space [ ]  aren't any elements larger than 10", function () {
```

Comment:
You did not refer to my comment from previous review regarding this test. I think this test should be removed.